src/components/date/date.js

```diff
@@ -500,14 +516,66 @@ class Date extends React.Component {
   }
 
   /**
+   * Updates the containerStyle state
+   *
+   * @method updateDatePickerPosition
+   * @return {Void}
+   */
+  updateDatePickerPosition = () => {
+    this.setState({ containerStyle: this.containerStyle });
+  }
+
+  /**
+   * Returns the bounding rect for the input
+   *
+   * @method getInputBoundingRect
+   * @return {Object}
+   */
+  getInputBoundingRect() {
+    return this._input.getBoundingClientRect();
+  }
+
+  /**
+   * Returns the style for the DayPicker container
+   *
+   * @method containerStyle
+   * @return {Object}
+   */
+  get containerStyle() {
+    const inputRect = this.getInputBoundingRect();
+    const offsetY = window.pageYOffset;
+    return {
+      left: inputRect.left,
+      top: inputRect.bottom + offsetY
+    };
+  }
+
+  /**
+   * Returns the props for the DayPicker container
+   *
+   * @method containerProps
+   * @return {Object}
+   */
+  get containerProps() {
+    return {
+      style: this.state.containerStyle,
+      onClick: this.handleWidgetClick
+    };
+  }
+
+  /**
    * Returns the DayPicker component
    *
    * @method renderDatePicker
    * @return {Object} JSX
    */
   renderDatePicker() {
     if (!this.state.open) { return null; }
-    return <DayPicker { ...this.datePickerProps } />;
+    return (
+      <Portal open>
```

Comment:
Do we just want the portal to deal with the open state. Rather than doing a check in the datepicker?

Reply:
@SeanArmstrong done